packages/fork-choice/src/protoArray/computeDeltas.ts

```diff
@@ -21,33 +21,38 @@ export function computeDeltas(
 ): number[] {
   const deltas = Array.from({length: indices.size}, () => 0);
```

Comment:
This can also be improved, it's faster to do a regular for loop than array.from

Reply:
Last time i checked regular for loop in the fastest way to populate an array. There should be benchmarks in the state transition package

Reply:
```
  array creation
Array.from(() => 0): 9.6100125 ms
    ✔ Array.from(() => 0)
Array.from().fill(0): 8.389225 ms
    ✔ Array.from().fill(0)
Array.from(): 15.170741699999999 ms
    ✔ Array.from()
new Array(): 0.8613583 ms
    ✔ new Array()
new Array(); for loop: 1.2474166 ms
    ✔ new Array(); for loop
```

@dapplion I'll do `new Array() and a for loop in` in https://github.com/ChainSafe/lodestar/pull/5882 👍

Reply:
Have you benchmark just pushing to an empty array?

Reply:
not yet, how important is it? in this case there are always more than 0 protonodes in forkchoice